chemprop/utils/utils.py

```diff
@@ -32,7 +32,7 @@ def items(cls) -> Iterator[tuple[str, str]]:
         return zip(cls.keys(), cls.values())
 
 
-def make_mol(smi: str, keep_h: bool, add_h: bool) -> Chem.Mol:
+def make_mol(smi: str, keep_h: bool, add_h: bool, ignore_chirality: bool) -> Chem.Mol:
```

Comment:
Since `make_mol` is part of the public API we should add this in a backwards compatible way:
```suggestion
def make_mol(smi: str, keep_h: bool, add_h: bool, ignore_chirality: bool = True) -> Chem.Mol:
```
I think this is sufficient, and won't break any thing else added here.

Reply:
Ah, ok. How about setting the default value to `False` instead? This would extend backward compatibility to the function's behavior as well.

Reply:
AH yes it should be False, good catch